Test for 0-add_integer.py

The module contains a function add_integer(a, b=98) that two arguments (integers to be added)

>>> a = __import__('0-add_integer').add_integer
>>> a(5, 3)
8

>>> a(-5, -3)
-8

>>> a(9, -3)
6

>>> a(5.2, 6)
11

>>> a(2)
100

>>> a("a", 5)
Traceback (most recent call last):
TypeError: a must be an integer


>>> a(5, "7")
Traceback (most recent call last):
TypeError: b must be an integer

>>> a(None)
Traceback (most recent call last):
TypeError: a must be an integer


>>> a(0, None)
Traceback (most recent call last):
TypeError: b must be an integer
